app/services/case_filter/case_date_range_filter_base.rb

```diff
@@ -0,0 +1,96 @@
+module CaseFilter
+  class CaseDateRangeFilterBase < CaseFilterBase
+
```

Comment:
As per the `CaseMultiChoicesFilterBase` would be good to discuss adding this via a mixin rather than a class :)